tools/docker-images/clp-execution-base-focal/setup-scripts/install-prebuilt-packages.sh

```diff
@@ -7,7 +7,7 @@ set -e
 set -u
 
 apt-get update
-DEBIAN_FRONTEND=noninteractive apt-get install -y \
+DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
   checkinstall \
   curl \
```

Comment:
we should add libssl-dev here unless we plan to remove the dependency in the short term?

Reply:
Good catch.